### PR TITLE
Add support for Azure Logging

### PR DESCRIPTION
**Pull request by burtbeckwith**
I skipped the Azure resource configuration section in the docs because there are several bits to configure. I'm finishing up the GDK guide and will create a Micronaut guide with the same info, and update the docs to reference that.

### CODE REVIEW

**Review by n0tl3ss — APPROVED** [review]
> Looks good to me. It would be nice if we can have a native test that would test the whole integration with Azure. :)

File: src/main/docs/guide/azureLogging.adoc [hunk 1]
```diff
@@ -0,0 +1,155 @@
+To use https://learn.microsoft.com/en-us/azure/azure-monitor/logs/data-platform-logs[Azure Monitor Logs], add the following dependency to your project:
+
+dependency:io.micronaut.azure:micronaut-azure-logging[]
+
+=== Create Azure Logging Resources
+
+TODO
```
Comment (graemerocher) [inline src/main/docs/guide/azureLogging.adoc]: address this TODO

File: src/main/docs/guide/azureLogging.adoc [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    <root level='INFO'>
+        <appender-ref ref='AZURE' />
+    </root>
+
+</configuration>
+----
+
+=== Browsing the logs
+
+TODO
```
Comment (graemerocher) [inline src/main/docs/guide/azureLogging.adoc]: address this TODO

**Review by graemerocher — APPROVED** [review]

File: azure-logging/build.gradle [hunk 3]
```diff
@@ -0,0 +1,28 @@
+plugins {
+    id 'io.micronaut.build.internal.azure-module'
+}
+
+dependencies {
+    annotationProcessor mn.micronaut.inject.java
```
Comment (sdelamo) [inline azure-logging/build.gradle]: This should be not necessary. The micronaut build plugins add it for you: 

```suggestion
```

File: azure-logging/build.gradle [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    api mnSerde.micronaut.serde.jackson
+    api libs.azure.monitor.ingestion
+    implementation mnLogging.logback.classic
+    implementation libs.logback.json.classic, {
+        exclude group: 'ch.qos.logback', module: 'logback-classic'
+    }
+
+    testAnnotationProcessor mn.micronaut.inject.java
+    testImplementation mn.micronaut.inject.groovy
+    testImplementation mnTest.micronaut.test.spock
```
Comment (sdelamo) [inline azure-logging/build.gradle]: This should not be necessary. The micronaut build plugins add it for you. 
```suggestion

```

**Review by sdelamo — COMMENTED** [review]
I would like to [split documentation into sections](https://github.com/micronaut-projects/micronaut-azure/pull/728)

**Review by sdelamo — APPROVED** [review]

**Pull request merged**